mozilla/fx-private-relay
# Add promo-blocking UI to React

# New feature description

This implements the promotional email blocking feature in React. It's implemented as a [slider](https://react-spectrum.adobe.com/react-aria/useSlider.html), so it can be focused with the keyboard, and then switch states using the arrow keys. Since sliders need to control a numeric value (which might get exposed through screen readers?), I used 0 (no blocking), 50 (only block promotional emails), and 100 (block everything).

# Screenshot (if applicable)

![image](https://user-images.githubusercontent.com/4251/160109532-99a54ea9-a258-4dc4-823b-a4a2d3bc0671.png)

![image](https://user-images.githubusercontent.com/4251/160109557-518dfc3b-b164-415c-96af-541eb3db88f0.png)

# How to test

Visit https://deploy-preview-1688--fx-relay-demo.netlify.app/?mockId=some, you should see the slider in the alias cards there.

# Checklist
